lib/Bio/EnsEMBL/DataCheck/Checks/IndividualType.pm

```diff
@@ -0,0 +1,88 @@
+=head1 LICENSE
+
+Copyright [2018-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::IndividualType;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::DataCheck::Test::DataCheck;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME        => 'IndividualType',
+  DESCRIPTION => 'Individuals have the correct type for each species',
+  GROUPS      => ['variation_import'],
+  DB_TYPES    => ['variation'],
+  TABLES      => ['individual'],
+};
+
+sub tests {
+  my ($self) = @_;
+
+  my $species = $self->dba->species; 
```

Comment:
As mentioned in the other PR, this should be `$self->species`, rather than `$self->dba-species`.
I appreciate that this is not at all obvious, I'll add a bit about it on confluence.